cmd/swarm-rafttool/dump.go

```diff
@@ -101,6 +101,27 @@ func dumpWAL(swarmdir, unlockKey string, start, end uint64) error {
 					return err
 				}
 
+				if redact {
+					// redact sensitive information
+					for _, act := range r.Action {
+						target := act.GetTarget()
+						switch actype := target.(type) {
+						case *api.StoreAction_Secret:
+							actype.Secret.Spec.Data = []byte("SECRET REDACTED")
+						case *api.StoreAction_Config:
+							actype.Config.Spec.Data = []byte("CONFIG REDACTED")
+						case *api.StoreAction_Task:
+							if container := actype.Task.Spec.GetContainer(); container != nil {
+								container.Env = []string{"ENVVARS REDACTED"}
+							}
+						case *api.StoreAction_Service:
+							if container := actype.Service.Spec.Task.GetContainer(); container != nil {
+								container.Env = []string{"ENVVARS REDACTED"}
+							}
+						}
+					}
+				}
```

Comment:
Would make sense to move this logic to a separate function and write a quick test to make sure that giving some input the output is redacted?